pkg/scm/git/git.go

```diff
@@ -34,7 +34,7 @@ type stiGit struct {
 	runner util.CommandRunner
 }
 
-var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/]+\.git)$`)
+var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/])`)
```

Comment:
If you don't include "$" at the end, then you can have any chars at the end of the string and they don't have to match your regex... like gabe@host:/test!@#.\***blah